### PR TITLE
Add integration tests 

### PR DESCRIPTION
**Pull request by deribaucourt**

### CODE REVIEW

File: client/tsconfig.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "strict": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+		"composite": true
```
Comment (idillon-sfl) [inline client/tsconfig.json:13]: indentation is weird here

File: client/package.json [hunk 2]
```diff
@@ -252,7 +252,6 @@
     }
   },
   "scripts": {
-    "test": "npm run test:grammar",
```
Comment (idillon-sfl) [inline client/package.json:255]: I think it was good. It is intended to have a way to launch all the tests of the client at the same time, even if for now there is only test:grammar. It also give us a way to uniformly call the tests of the client and the server just with "npm test --workspaces" (thought the server has no tests for the moment).

File: package.json [hunk 3]
```diff
@@ -27,16 +27,21 @@
 		"watch:client": "tsc -w -p client/tsconfig.json",
 		"compile:server": "tsc -p server/tsconfig.json",
 		"watch:server": "tsc -w -p server/tsconfig.json",
-		"clean:server": "rm -fr ./server/node_modules && rm -fr ./server/out",
-		"clean:client": "rm -fr ./client/node_modules && rm -fr ./client/out",
-		"clean": "npm run clean:server && npm run clean:client && rm -fr node_modules",
+		"clean:server": "rm -fr ./server/node_modules && rm -fr ./server/out && rm -rf server/tsconfig.tsbuildinfo && rm -fr server/.vscode-test",
```
Comment (idillon-sfl) [inline package.json:30]: do you think writing down `rm -rf` just once could make this much more readable?
```suggestion
		"clean:server": "rm -fr ./server/node_modules ./server/out server/tsconfig.tsbuildinfo server/.vscode-test",
```

**Review by idillon-sfl — APPROVED** [review]
looks cool!

File: client/tsconfig.json (hunk 1 above)
Comment (deribaucourt) [reply client/tsconfig.json:13]: Done

File: package.json (hunk 3 above)
Comment (deribaucourt) [reply package.json:30]: Done

File: client/package.json (hunk 2 above)
Comment (deribaucourt) [reply client/package.json:255]: Done

File: src/integration-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Integration tests
```
[the remark this answers is not in the document]
Comment (deribaucourt) [reply src/integration-tests/README.md]: Done

**Comment by deribaucourt** [comment]
Thanks for your suggestions!

**Pull request merged**